src/visualizer.js

```diff
@@ -1,5 +1,28 @@
 'use strict';
 
+Room.prototype.myRoomDatasDraw = function() {
+
+  let color = (coeff) => `rgb(${Math.floor(-(coeff - 1) * 255)},${Math.floor(coeff * 255)},0)`;
+  if (this.memory.energyStats) {
+    this.visual.text(`Energie Average : ${this.memory.energyStats.average}`, 5, 0.5, {
```

Comment:
When used for the first time (less than ``ticksUntilHelpCheck`` tick), ``energyStats.average`` is undefined.